go.mod

```diff
@@ -42,7 +42,8 @@ require (
 
 replace (
 	github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v38.2.0+incompatible
-	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.4.0+incompatible
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.1-0.20201007172425-bf140ad4ab62+incompatible
+	github.com/Azure/go-autorest/autorest/adal => github.com/Azure/go-autorest/autorest/adal v0.9.6-0.20201007172425-bf140ad4ab62
```

Comment:
do we need to open issues with `go-containerregistry` & `k8s-pkg-credentialprovider` to bump go-autorest so we can drop the `replace` here or will this shake out on its own when they next release?

Reply:
Imma go have a look, thanks for the hint.